src/types/AuthenticationMiddlewareOptions.ts

```diff
@@ -1,13 +1,14 @@
 import { NextFunction, Request, Response } from 'express';
-import { Logger } from '.';
+import { Logger, GetTokenInfo } from '.';
 
 type onNotAuthenticatedHandler = (request: Request, resonse: Response, next: NextFunction, logger: Logger) => void;
 
-type AuthenticationMiddlewareOptions = {
+type AuthenticationMiddlewareOptions<T = {}> = {
```

Comment:
Is this type parameter fine? I'm not an expert of TypeScript...

Reply:
I think you can remove the generic type annotation here if you add it to `GetTokenInfo` (see next comment).